src/apic-extension/azext_apic_extension/command_patches.py

```diff
@@ -519,3 +543,199 @@ def pre_operations(self):
             "region_name": args.region_name,
             "msi_resource_id": args.msi_resource_id
         }
+
+
+@register_command(
+    "apic import apim",
+    is_preview=True,
```

Comment:
I'm not sure whether we expect to set this as preview, since the original command is in GA status. Let's confirm with PM.